sdlf-team/scripts/template-team-repos.yaml

```diff
@@ -126,4 +127,58 @@ Resources:
       Targets:
         - Arn: !Sub arn:aws:events:${AWS::Region}:${pChildAccountId}:event-bus/default
           Id: !Sub sdlf-cicd-team-codecommit-${pEnvironment}-${pTeamName}
-          RoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/sdlf-cicd-foundations-eventbus-${pEnvironment}
\ No newline at end of file
+          RoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/sdlf-cicd-foundations-eventbus-${pEnvironment}
+  rSNSTopic:
+    Type: "AWS::SNS::Topic"
+    Properties:
+      TopicName: !Sub "sdlf-repo-notifications-${pEnvironment}-${pTeamName}"
+      Subscription:
+        - Endpoint: !Ref pSNSNotificationsEmail
+          Protocol: "email"
+  rSNSTopicPolicy:
+    Type: "AWS::SNS::TopicPolicy"
+    Properties:
+      PolicyDocument:
+        Id: !Sub "sdlf-${pTeamName}-notifications"
+        Version: "2012-10-17"
+        Statement:
+          - Sid: !Sub "sdlf-${pTeamName}-notifications"
+            Effect: Allow
+            Principal:
+              Service: "cloudwatch.amazonaws.com"
+            Action: "sns:Publish"
+            Resource: !Ref rSNSTopic
+      Topics:
+        - !Ref rSNSTopic
```

Comment:
Finally this section would have to be entirely replaced with the below:
```
  rSNSTopic:
    Type: AWS::SNS::Topic
    Properties:
      TopicName: !Sub sdlf-repo-notifications-${pEnvironment}-${pTeamName}

  rSNSTopicPolicy:
    Type: AWS::SNS::TopicPolicy
    Properties:
      PolicyDocument:
        Id: !Sub sdlf-${pTeamName}-notifications
        Version: 2012-10-17
        Statement:
          - Sid: !Sub sdlf-${pTeamName}-notifications
            Effect: Allow
            Principal:
              Service: cloudwatch.amazonaws.com
            Action: sns:Publish
            Resource: !Ref rSNSTopic
      Topics:
        - !Ref rSNSTopic
  
  rSNSTopicSubscription:
    Type: Custom::SNSSubscription
    Properties:
      ServiceToken: !ImportValue sdlf-sns-topic-subscription-lambda-arn
      TeamName: !Sub ${pTeamName}
      TopicArn: !Ref rSNSTopic
      SubscriptionEndpoints: !Ref pSNSNotificationsEmail
      SubscriptionProtocol: email
```
Notice the custom subscription resource

Reply:
Thanks for the feedback. I'll be adding this changes